toc.json

```diff
@@ -268,6 +268,11 @@
           "type": "item",
           "title": "Public Certificates",
           "uri": "docs/webhooks/08-Certificates.md"
+        },
+        {
+          "type": "item",
+          "title": "Safelist IPs",
```

Comment:
Should we call this page "Webhook IPs" since that's the primary purpose of the page? "Safelisting" is something you might do with those IPs...

Reply:
yep